packages/apps/galaxpay/src/galaxpay-list-payments.ts

```diff
@@ -0,0 +1,128 @@
+  // const initialTotalAmount = amount.total;
+
+  const configApp = {
+    ...application.data,
+    ...application.hidden_data,
+  } as GalaxpayApp;
+
+  // setup basic required response object
+  const response: ListPaymentsResponse = {
+    payment_gateways: [],
+  };
+
+  if (!configApp.galaxpay_id || !configApp.galaxpay_hash) {
+    return responseError(
+      409,
+      'NO_GALAXPAY_KEYS',
+      'GalaxPay ID e/ou GalaxPay Hash da API indefinido(s) (lojista deve configurar o aplicativo)',
+    );
+  }
+
+  // common payment methods data
+  const intermediator = {
+    name: 'GalaxPay',
+    link: `https://api.${isSandbox ? 'sandbox.cloud.' : ''}galaxpay.com.br/v2`,
+    code: 'galaxpay_app',
+  };
+
+  const paymentTypes: Gateway['type'][] = [];
+  if (configApp.plans) {
+    paymentTypes.push('recurrence');
+  }
+
+  // setup payment gateway objects
+  const plans = handleGateway(configApp);
+
+  plans.forEach((plan) => {
+    ['credit_card', 'banking_billet', 'pix'].forEach((paymentMethod) => {
+      paymentTypes.forEach((type) => {
+        const methodConfig = configApp[paymentMethod] || {};
+        const methodMinAmount = methodConfig.min_amount || 0;
+        if (!methodConfig.disable && (amount.total && methodMinAmount <= amount.total)) {
+          logger.log('> Plan ', plan.periodicity);
+
+          const isCreditCard = paymentMethod === 'credit_card';
+          const isPix = paymentMethod === 'pix';
+          let { label } = methodConfig;
+          if (!label) {
+            if (isCreditCard) {
+              label = 'Cartão de crédito';
+            } else {
+              label = isPix ? 'PIX' : 'Boleto bancário';
+            }
+          }
+
+          const periodicity = parsePeriodicityToEcom(plan.periodicity);
+          const planName = plan.label ? plan.label : 'Plano';
+
+          if (type === 'recurrence' && planName) {
+            label = `${planName} ${periodicity} ${label}`;
+          }
+          const gateway: Gateway = {
+            label,
+            icon: methodConfig.icon,
+            text: methodConfig.text,
+            payment_method: {
+              code: isPix ? 'account_deposit' : paymentMethod as CodePaymentMethod, // pix is defined payment method outher
+              name: `${label} - ${intermediator.name}`,
+            },
+            type,
+            intermediator,
+          };
+
+          if (isCreditCard) {
+            if (!gateway.icon) {
+              // Alternative solution
+              gateway.icon = 'https://ecom-galaxpay.web.app/credit-card.png';
+              // TODO:
+            }
+            // https://docs.galaxpay.com.br/tokenizacao-cartao-js
+            gateway.js_client = {
+              script_uri: 'https://js.galaxpay.com.br/checkout.min.js',
+              onload_expression: `window._galaxPayPublicToken="${configApp.galaxpay_public_token}";
+              window._galaxPaySandbox="${isSandbox}";
```

Comment:
```suggestion
              window._galaxPaySandbox=${isSandbox};
```

`"false"` é positivo

Reply:
Ou  você checa `_galaxPaySandbox === "true"` ? (string, não booleano)